src/frontend/app/Controller/CodeController.js

```diff
@@ -46,15 +46,14 @@ export default class CodeController {
       zIndex: 3001,
       callback: function (key, options) {
         var m = "clicked: " + key;
-        window.console && console.log(m) || alert(m);
+        alert(m);
```

Comment:
Are these alert messages necessary? Does it show an error to the user or is it for debug purposes?